app/routes/events/components/EventEditor/EditorSection/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Flex, Icon } from '@webkom/lego-bricks';
+import cx from 'classnames';
+import { useState } from 'react';
+import styles from '../EventEditor.css';
+import type { PropsWithChildren } from 'react';
+
+type Props = {
+  title: string;
+  collapsible?: boolean;
+  initiallyExpanded?: boolean;
+};
+
+const EditorSection: React.FC<PropsWithChildren<Props>> = ({
```

Comment:
Now that I think of it, an [accordion component](https://ui.shadcn.com/docs/components/accordion) would probably be a nice addition to lego-bricks. I feel like I know a couple of other places where it can be used later

Reply:
Absolutely, currently used in at least polls and readme - and the functionality is very much generalizable (and the current implementation, although recently improved - still has a bit of room for improvement)

Reply:
Don't think I'll do that in this PR tho, so I'm resolving this thread for now